### ISSUE TITLE
Rotate Swastick.

### ISSUE THREAD

**Issue by RohitSaxenaJii** [RohitSaxenaJii/swastick-question#1]
When we get output which is a swastick, make it rotate 360 degree twice. Do it..

**Comment by void-57** [comment]
Assign me@RohitSaxenaJii

**Comment by mrsaxena77** [comment]
yes I can do this. By using translate concept in css. please assign this to me.

**Comment by SunnyRajpoot2511** [comment]
I can Also do this. Please allow me too. so that i also raise the count of my PRs.
please.

**Comment by Mukesh1960** [comment]
Please assign me too sir.
I will contribute in all repositories.

**Comment by Mamta1401** [comment]
hi sir,
please also assign this to me.
my work will be so different and effective.
and I also want to contribute in all of your repositories.
please assign me too sir.

**Comment by thecrusader25225** [comment]
can i work on it? if so, please assign me.
